test/addons/rbd-mirror/start

```diff
@@ -171,6 +172,8 @@ wait_until_pool_mirroring_is_healthy(cluster2)
 deploy_vrc_sample(cluster1)
 deploy_vrc_sample(cluster2)
 
+time.sleep(30)
```

Comment:
Interesting! maybe we can wait instead until the blocklist entries disappear or some timeout expires?

Reply:
Or - since we deploy the vrc and it probably take some time until it is processed by csi-addons and ceph - maybe wait for the resource until it is ready?

It seems that our check for osd blocklist is not correct since entries can be created by interacting with ceph, or maybe there is a bug in ceph. But lets see if this happens when we wait for the vrc.

Reply:
The blocklist checks are removed by #1237 - they are a common source of failures, and the rbd-mirroring tests works fine with the blocklists.